build.sh

```diff
@@ -101,21 +94,36 @@ if [[ $CLEAN_BUILD == 'YES' ]]; then
 fi
 mkdir -p ${BUILD_DIR} && cd ${BUILD_DIR}
 
-# If INSTALL_PREFIX is not empty; install at INSTALL_PREFIX
-[[ -n "${INSTALL_PREFIX:-}" ]] && CMAKE_OPTS+=" -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX}"
-
-# activate tests based on if this is cloned within the global-workflow
-WORKFLOW_BUILD=${WORKFLOW_BUILD:-"OFF"}
+# Set WORKFLOW_TESTS as CMake option
 CMAKE_OPTS+=" -DWORKFLOW_TESTS=${WORKFLOW_TESTS:-${WORKFLOW_BUILD}}"
 
-# Link MOM6 and Icepack in SOCA to submodules in the UFS repo
 if [[ $WORKFLOW_BUILD == 'ON' ]]; then
+  # Set INSTALL_PREFIX to the Global Workflow home directory
+  [ -d "$HOMEgfs" ] || { echo "Error: $HOMEgfs does not exist" >&2; exit 1; }
+  INSTALL_PREFIX=$HOMEgfs
+
+  # If $HOMEgfs/bin already exists, move it to a temporary directory
+  # This is to avoid conflicts with the GDASApp executables that will be installed
+  if [ -d "$HOMEgfs/bin" ]; then
+    echo "Warning: $HOMEgfs/bin already exists, moving it to $HOMEgfs/bin_temp"
+    [ -d "$HOMEgfs/bin_temp" ] || { echo "Error: $HOMEgfs/bin_temp already exists" >&2; exit 1; }
+    mv $HOMEgfs/bin $HOMEgfs/bin_temp
```

Comment:
Should not move the contents of `$HOMEgfs`.
The purpose of this script is to build the executables, libraries, etc. and drop them under `$HOMEgfs/exec`  or `INSTALL_PREFIX/exec`.